vcluster/configure/vcluster-yaml/sync/from-host/custom-resources.mdx

```diff
@@ -34,6 +31,151 @@ sync:
         scope: Cluster
 ```
 
+## Namespaced scope custom resource definitions
+By default, this is turned off.
```

Comment:
Perhaps this isn't necessary ? I wouldn't assume this is on by default since it's not standard.

Reply:
I was following the convention of all pages under sync / from host - they all say at the top whether it is enabled / disabled by default. For example, events are enabled by default.